src/js/menu/menu-button.js

```diff
@@ -37,6 +37,16 @@ class MenuButton extends ClickableComponent {
   }
 
   /**
+   * Hide the menu if the number of items is less than or equal to this threshold. This defaults
```

Comment:
Ultimately I'd love to see each menu component actually set their threshold as they add items to the menu which can be hidden (title, off button, caption settings button, etc), but this looks like it will do for now, and covers both #3890 and #3994.